mypy/build.py

```diff
@@ -508,6 +508,9 @@ def __init__(self, data_dir: str,
         self.modules = {}  # type: Dict[str, MypyFile]
         self.missing_modules = set()  # type: Set[str]
         self.fg_deps_meta = {}  # type: Dict[str, FgDepMeta]
+        # Always convert the plugin to a ChainedPlugin so that it can be manipulated if needed
+        if not isinstance(plugin, ChainedPlugin):
+            plugin = ChainedPlugin(options, [plugin])
```

Comment:
Yeah, we could. An earlier version modified load_plugins and updated a call site in the tests. I switched it to this way because it seemed cleaner: the need for a ChainedPlugin is an internal detail that doesn't need to be exposed in the constructor.